frontend/translations.js

```diff
@@ -6,6 +6,7 @@ module.exports = {
   SendAmountIsNotPositive: () => 'Invalid format: Amount has to be a positive number',
   SendAmountInsufficientFunds: ({balance}) =>
     `Insufficient funds for the transaction. Your balance is ${printAda(balance)} ADA.`,
+  SendAmountCantSendAllFunds: () => 'Insufficient balance to send all funds',
```

Comment:
Suggesting to rewrite to something like `Sending funds is not possible because there is no balance for paying tx fee`,